lib/app/features/landing/landing.dart

```diff
@@ -0,0 +1,93 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:flutter_modular/flutter_modular.dart';
+import 'package:score_live/app/custom_widgets/custom_app_bar.dart';
+import 'package:score_live/app/features/landing/cubit/landing_cubit.dart';
+import 'package:score_live/app/features/landing/landing_module.dart';
+
+class LandingPage extends StatefulWidget {
+  const LandingPage({
+    super.key,
+  });
+
+  @override
+  State<LandingPage> createState() => _LandingPageState();
+}
+
+@override
+void dispose() {
+  Modular.dispose<LandingCubit>();
+}
+
+class _LandingPageState extends State<LandingPage> {
+  @override
+  Widget build(BuildContext context) {
+    LandingCubit landingCubit = Modular.get<LandingCubit>();
+    return BlocProvider(
+      create: (context) => landingCubit,
+      child: BlocConsumer<LandingCubit, LandingState>(
+        listener: (context, state) {
```

Comment:
You can move params `(context, state)` to method and call it `_landingScreenListener`